web/src/services/Assertion.service.ts

```diff
@@ -1,11 +1,46 @@
 import countBy from 'lodash/countBy';
 import uniq from 'lodash/uniq';
 
-import {TAssertionResult, ICheckResult, TRawAssertionResult} from 'types/Assertion.types';
+import {ICheckResult, TAssertionResult, TRawAssertionResult} from 'types/Assertion.types';
+import {durationRegExp} from '../constants/Common.constants';
+import {Attributes} from '../constants/SpanAttribute.constants';
+
+const isNumeric = (num: string): boolean => /^-?\d+(?:\.\d+)?$/.test(num);
+const isNumericTime = (num: string): boolean => durationRegExp.test(num);
 
 const AssertionService = () => ({
+  extractExpectedString(input?: string): string | undefined {
+    if (!input) return input;
```

Comment:
https://github.com/kubeshop/tracetest/issues/1160

This method solves this issue

Reply:
It is a requirement from the backend, @xoscar. Due to expression support if we don't wrap strings with quotes, it is ambiguous for the backend if that should be an attribute or a string. Example:

```
http.status_text = ok
```

In this case, `ok` can be an attribute from that span or a string. Due to how the parser works, this will be considered an attribute from the span. And probably the span doesn't have that attribute so it will be resolved as an empty string.

If you want to compare that as a string, you have to send:

```
http.status_text = "ok"
```